README.md

```diff
@@ -115,42 +121,42 @@ It is also possible to Download a font from
 
 ### Print text
 
-```dymoprint MyText```
+```labelle MyText```
 
 Multilines will be generated on whitespace
 
-```dymoprint MyLine MySecondLine # Will print two Lines```
+```labelle MyLine MySecondLine # Will print two Lines```
 
 If you want whitespaces just enclose in " "
 
-```dymoprint "prints a single line"```
+```labelle "prints a single line"```
 
 ### Print QRCodes and Barcodes
 
-```dymoprint --help```
+```labelle --help```
 
 ### Print Codes and Text
 
 Just add a text after your qr or barcode text
 
-```dymoprint -qr "QR Content" "Cleartext printed"```
+```labelle -qr "QR Content" "Cleartext printed"```
 
 ### Picture printing
 
 Any picture with JPEG standard may be printed. Beware it will be downsized to tape.
 
-```dymoprint -p mypic.jpg ""```
+```labelle -p mypic.jpg ""```
 
 Take care of the trailing "" - you may enter text here which gets printed in front of the image
 
 ## GUI
 
-### Run DymoPrint GUI
+### Run Labelle GUI
 
-```dymoprint_gui```
+```labelle-gui```
```

Comment:
Thank you so much for changing underscore with dash. It was rather annoying, and I planned on changing this after migration :+1: